routergroup.go

```diff
@@ -185,11 +185,19 @@ func (group *RouterGroup) StaticFS(relativePath string, fs http.FileSystem) IRou
 func (group *RouterGroup) createStaticHandler(relativePath string, fs http.FileSystem) HandlerFunc {
 	absolutePath := group.calculateAbsolutePath(relativePath)
 	fileServer := http.StripPrefix(absolutePath, http.FileServer(fs))
-	_, nolisting := fs.(*onlyfilesFS)
+
```

Comment:
why remove the line? can you explain? thanks!

Reply:
I removed it because it wasn't required, I could directly use fs.Open() to get both if it exists (no need to use fs.(*onlyfilesFS)) and if it's accessible. Two function calls (half-reduntant) wouldn't really be efficient.

Reply:
Ok, thanks!